### PR TITLE
Translate docs/node-apis

### PR DESCRIPTION
**Pull request by gauravgupta45**

### CODE REVIEW

File: docs/docs/node-apis.md [hunk 1]
```diff
@@ -1,15 +1,16 @@
 ---
 title: Gatsby Node APIs
-description: Documentation on Node APIs used in Gatsby build process for common uses like creating pages
+description: Documentation sur les API Node utilisées dans le processus de construction de Gatsby pour des utilisations courantes comme la création de pages
 jsdoc: ["gatsby/src/utils/api-node-docs.js"]
 apiCalls: NodeAPI
 ---
 
-Gatsby gives plugins and site builders many APIs for controlling your site's data in the GraphQL data layer.
+
```
Comment (SamuelAlev) [inline docs/docs/node-apis.md]:
```suggestion
```

Extra line to be removed

File: docs/docs/node-apis.md [hunk 2]
```diff
@@ -26,8 +27,8 @@ exports.createPages = (_, pluginOptions, cb) => {
 }
 ```
 
-If your plugin does not do async work, you can just return directly.
+Si votre plugin ne fait pas de travail asynchrone, vous pouvez simplement retourner directement.
 
 ## Usage
```
Comment (SamuelAlev) [inline docs/docs/node-apis.md]:
```suggestion
## Utilisation
```

File: docs/docs/node-apis.md [hunk 3]
```diff
@@ -1,15 +1,16 @@
...
+description: Documentation sur les API Node utilisées dans le processus de construction de Gatsby pour des utilisations courantes comme la création de pages
 jsdoc: ["gatsby/src/utils/api-node-docs.js"]
 apiCalls: NodeAPI
 ---
 
-Gatsby gives plugins and site builders many APIs for controlling your site's data in the GraphQL data layer.
+
+Gatsby offre aux plugins et aux constructeurs de sites de nombreuses API pour contrôler les données de votre site dans la couche de données GraphQL.
 
 ## Async plugins
```
Comment (SamuelAlev) [inline docs/docs/node-apis.md]:
```suggestion
## Plugins asynchrone
```

**Review by SamuelAlev — CHANGES_REQUESTED** [review]
Hello,
Thank you for your work!
However, you have not translated the title.

Please check your PRs before submitting them.

You will find the Contributing guide over there: https://www.gatsbyjs.org/contributing/

**Comment by gauravgupta45** [comment]
Titles have been translated

**Review by SamuelAlev — APPROVED** [review]

**Pull request merged**